commands/nb.py

```
@@ -1,104 +1 @@
-        print("⚠️  Warning: Failed to set upstream tracking (this is usually okay)")
-        print(f"stderr: {stderr}")
-
-    print(f"✅ Successfully created and switched to branch: {branch_name}")
-    print("📋 Branch is based on latest main and ready for development")
-
-    return 0
-
-
-if __name__ == "__main__":
-    sys.exit(main())
+newbranch.py
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)